### ISSUE TITLE
Tutorial to train a cross-language model with sentencepiece

### ISSUE THREAD

**Issue by loretoparisi** [google/sentencepiece#285]
It would be worth to provide a tutorial how to train a simple cross-language classification model using sentencepiece. Supposed to have a given training set and have chosen a model (let'say a simple Word2Vec plus softmax or a LSTM model, etc), how to use the created sentencepiece model (vocabulary/codes) to feed this model for train and inference?

**Comment by ebuildy** [comment]
(in my understanding) sentencepiece is a tokenizer, you can use it to pre-process your corpus, before the embedding step.

**Comment by loretoparisi** [comment]
@ebuildy yes it is, Basically it's a BPE tokenizer, so it turns sentences into codes, and map the codes to words into a vocabulary (actually not words, more like subwords).
I know how to use it

```

TURN NEWLINE INTO USER DEFINED
awk '{gsub(/\\n/,"<n>")}1' /root/my_dataset.csv > /root/test_dataset.csv

TRAIN WITH USER DEFINED NEWLINE
spm_train --max_sentence_length=12000 --user_defined_symbols='<n>' --model_prefix=/root/model --vocab_size=32000 --character_coverage=1.0 --model_type=bpe --input=/root/test_dataset.csv

ENCODE
echo "Are you eating cereal or oatmeal?" | spm_encode --model=/root/test_spm.model --output_format=piece
▁ H e llo ▁ W or ld

DECODE
echo "▁ N ew ▁ Y ork" | spm_decode --model=test_spm.model --input_format=piece
```

The point is that, assumed I want to use it on a let's say BERT (that uses WordPiece instead), I don't know how to handle the inputs to feed the model.

**Comment by AES0007** [comment]
I followed the instructions but all tokenization values is exactly the same for every sentence...any thoughts?

**Comment by taku910** [comment]
We are going to close this issue soon as the issue is deprecated.
